tests/lib/rules/forbid-component-props.js

```diff
@@ -166,21 +166,6 @@ ruleTester.run('forbid-component-props', rule, {
         },
       ],
     },
-    {
-      code: `
-        const item = (<Foo className="foo" />);
-      `,
-      options: [
-        {
-          forbid: [
-            {
-              propName: 'className',
-              disallowedFor: ['ReactModal'],
-            },
-          ],
-        },
-      ],
-    },
```

Comment:
Again two same in a row:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/forbid-component-props.js#L154-L183